src/sap/fhir/model/r4/FHIRFilterOperatorUtils.js

```diff
@@ -71,19 +75,19 @@ sap.ui.define(["sap/fhir/model/r4/FHIRFilterOperator", "sap/fhir/model/r4/FHIRFi
 	 * @public
 	 * @since 1.0.0
 	 */
-	FHIRFilterOperatorUtils.isSearchParameterPrefixable = function(oFilter) {
+	FHIRFilterOperatorUtils.isSearchParameterPrefixable = function (oFilter) {
 		return !(typeof oFilter.oValue1 === "string" || Array.isArray(oFilter.oValue1)) || oFilter.sValueType === FHIRFilterType.date || !isNaN(oFilter.oValue1);
 	};
 
 	/**
-	 * Parses the JS filter value to an FHIR filter value
+	 * Parses the JS filter value to a FHIR filter value
 	 *
 	 * @param {any} oValue The value of a filter object
 	 * @returns {string} Formatted FHIR filter value
-	 * @public
+	 * @protected
```

Comment:
Oh I see.... which should stay compatible. If the method was already public it has to stay public.

Reply:
done